bindings/js/src/wasm_wrapper.c

```
@@ -1,46 +1,50 @@
 #include <cimg2num.h>
 #include <emscripten/emscripten.h>
 
-EMSCRIPTEN_KEEPALIVE void gaussian_blur_fft(uint8_t *image, size_t width, size_t height,
-                                            double sigma) {
+EMSCRIPTEN_KEEPALIVE void
+gaussian_blur_fft(uint8_t* image, size_t width, size_t height, double sigma) {
```

Cppcheck (2.20.0)

[style] The function 'gaussian_blur_fft' is never used.

(unusedFunction)